src/inc/crsttypes.h

```diff
@@ -1,6 +1,8 @@
+//
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
+//
```

Comment:
Probably want to leave the license header as it was, I assume we have some source scanner that will complain eventually if the header isn't exactly right